linuxNew/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java

```diff
@@ -44,6 +44,7 @@
 		 */
 		return Stream.of(
 			Arguments.of("amazonlinux", "2"),
+			Arguments.of("centos", "6"),
```

Comment:
Looks like this doesn't work.  Test job failed. https://github.com/adoptium/installer/runs/4704365456?check_suite_focus=true

Reply:
Should also add the test for rhel 6. Does the rhel image 6 exist in DockerHub?

Reply:
no unfortunately not